index.js

```diff
@@ -0,0 +1,88 @@
+// import _ from 'lodash';
```

Comment:
Kindly remove all commented lines of old code. It will make your code a lot cleaner.